### ISSUE TITLE
Tortuga: Cannot transfer just a domain to another WordPress.com user

### ISSUE THREAD

**Issue by fabianapsimoes** [Automattic/wp-calypso#12102]
Originally reported by @designsimply in #30-a8c-domain-search:

Steps to reproduce:

1. Purchase just a domain for a site on WordPress.com.
1. Go to Settings > Transfer Domain > Transfer to another user.
1. Try to add a user to transfer the domain to.

<strong>Result:</strong> I don't have access to the page where I can add a user, so I cannot get to a point where I can transfer the domain to someone else.

Video: <a href="https://cloudup.com/cEyhjQLi62f">33s</a>

![screen shot 2017-03-13 at mon mar 13 7 21 19 pm](https://cloud.githubusercontent.com/assets/1119271/23881794/7259ea2c-0822-11e7-8d20-1fe92e4daaac.png)
<sup>Seen at https://wordpress.com/domains/manage/tellthislikeitis.blog/transfer/other-user/tellthislikeitis.blog using Safari 10.0.3 on Mac OS X 10.12.3.</sup>

**Comment by fabianapsimoes** [comment]
@breezyskies I'd say we should probably just hide that functionality altogether for domain-only users?

**Comment by breezyskies** [comment]
> I'd say we should probably just hide that functionality altogether for domain-only users?

Yep, that my thought as well.

**Comment by designsimply** [comment]
Is the option to transfer just a domain to another user something you would consider adding later? It feels like a good option for users to have if possible.

**Comment by breezyskies** [comment]
The "transfer to another user" is specifically for other users on the same site (you can't transfer to _any_ user on WordPress.com). For example, a designer or dev might have purchased the domain for their client and need to transfer ownership once the site is done and setup. This would be available as soon as the user adds a site -- until they have a site, the scenario doesn't make as much sense, and I would consider it pretty low priority. 

(Tbh, I'm not convinced we should include this type of functionality in the UI at all. It overcomplicates and already complex UI for domains, for something that a small number of users would use very rarely.)

**Comment by designsimply** [comment]
I would consider it a low-volume-high-pain-point for the cases where it does come up (i.e. agency or individual purchases a domain on behalf of another individual and then would like to transfer ownership prior to setting up a site). It is something I would expect to be able to do through a registrar but not necessarily a dedicated site hosting provider such as WordPress.com, and I agree that setting up a site is an okay workaround for the problem.
